api/src/cli/xml_generation_cli.py

```diff
@@ -34,7 +34,7 @@
 @click.option(
     "--form",
     default="SF424_4_0",
-    help="Form name/version (e.g., SF424_4_0, SF424A, SFLLL_2_0). Default: SF424_4_0",
+    help="Form name/version (e.g., SF424_4_0, SF424A, SFLLL_2_0, CD511). Default: SF424_4_0",
```

Comment:
Could use the mapping function you setup to adjust this make it a list of choices. If there is an error/`--help` is added, it'll output that list, and it dynamically changes without effort. No need to handle it in this PR, maybe the next one.

```py
type=click.Choice(_build_xml_form_map.keys()),
```